routes/contributions.js

```diff
@@ -0,0 +1,30 @@
+const express = require('express')
+const router = express.Router()
+const contributionsController = require('../controllers/contributionsController')
+
+/**
+ * @swagger
+ * /contributions/:username:
+ *   get:
+ *     summary: Used to get all the contributions of user
+ *     tags:
+ *       - Contributions
+ *     responses:
+ *       200:
+ *         description: Return contributions
+ *         content:
+ *           application/json:
+ *             schema:
+ *               $ref: '#/components/schemas/contributions'
+ *       503:
+ *         description: serverUnavailable
+ *         content:
+ *           application/json:
+ *             schema:
+ *               $ref: '#/components/schemas/errors/serverUnavailable'
+ */
+router
```

Comment:
`router.get('/:username', contributionsController.getUserContributions)` ?

Reply:
Just tried it :smile: